packages/airplane/internal/api/fetcher.ts

```diff
@@ -84,7 +85,11 @@ export class Fetcher {
     );
   }
 
-  async get<Output = unknown>(path: string, params?: Record<string, unknown>): Promise<Output> {
+  async get<Output = unknown>(
+    path: string,
+    params?: Record<string, unknown>,
+    enableTracing?: boolean
```

Comment:
Make this an options object. This makes it more clear what `true` means. It also allows us to introduce new options.

```suggestion
    opts: { enableTracing?: boolean } = {}
```

Reply:
Done.